src/parser/jobs/mnk/modules/evaluators/BlitzEvaluator.tsx

```diff
@@ -119,16 +114,16 @@ export class BlitzEvaluator implements WindowEvaluator {
 	}
 
 	private expectedBlitzes(window: HistoryEntry<EvaluatedAction[]>): number {
-		const priorCasts = this.pbCasts.filter(time => time <= window.start)
-		const priorRegens = this.rechargeTimes.filter(time =>
-			time <= (window.end ?? window.start) - END_OF_WINDOW_TOLERANCE
-		)
+		// We expect the player to burn off PB charges that were available when the window began, and any that became available with enough time before the window ended
+		const chargesAvailable = (_.last(this.pbChargeHistory.filter(entry => entry.timestamp < window.start))?.current ?? this.maxCharges) +
+			(this.pbChargeHistory.some(entry => entry.timestamp > window.start && entry.timestamp <= (window.end ?? window.start) - END_OF_WINDOW_TOLERANCE) ? 1 : 0)
```

Comment:
There seems to be a bug with end of fight tolerance here. In this situation my PB charge came up ~1 GCD before the fight ended, but it expected a blitz:

![image](https://github.com/xivanalysis/xivanalysis/assets/65056303/439a4483-2663-467a-a3c0-b5ca12222712)

Live version:

![image](https://github.com/xivanalysis/xivanalysis/assets/65056303/6b2530c7-8005-45b0-a2aa-4cf8ed468094)

https://xivanalysis.com/fflogs/FwZtxbynRKVTP94g/1/8

Reply:
Should be fixed now!